components/elm/cime_config/config_component.xml

```diff
@@ -217,34 +217,35 @@
       <value compset="4804.*_ELM.*_CISM1">off</value>
       <value compset="_ELM.+MALI"        >on</value>
     </values>
-    <group>run_component_clm</group>
+    <group>run_component_elm</group>
     <file>env_run.xml</file>
     <desc>Flag to the ELM build-namelist command to force ELM to do a
       cold start (finidat will be set to blanks).
       A value of on forces the model to spin up from a cold-start
       (arbitrary initial conditions). Setting this value in the xml file will take
-      precedence over any settings for finidat in the $CASEROOT/user_clm_clm file.</desc>
+      precedence over any settings for finidat in the $CASEROOT/user_elm file.</desc>
   </entry>
 
   <description>
-    <desc compset="_ELM"               >clm4.5 physics:</desc>
-    <desc compset="_ELM%[^_]*SP"       >clm4.5 Satellite phenology:</desc>
-    <desc compset="_ELM%[^_]*SPBC"     >clm4.5 Satellite phenology with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CN"       >clm4.5 cn:</desc>
-    <desc compset="_ELM%[^_]*CNDV"     >clm4.5 cn with dynamic vegetation:</desc>
-    <desc compset="_ELM%[^_]*BGC"      >clm4.5 bgc (cn and methane):</desc>
-    <desc compset="_ELM%[^_]*CROP"     >clm4.5 prognostic crop:</desc>
-    <desc compset="_ELM%[^_]*VIC"      >clm4.5 vic hydrology:</desc>
-    <desc compset="_ELM%[^_]*FATES"    >clm4.5 FATES (Functionally Assembled Terr. Ecosystem Simulator): (experimental)</desc>
-    <desc compset="_ELM%[^_]*BGCDV"    >clm4.5 BGC (CN with vertically resolved soil BGC, based on Century with Methane) with dynamic veg</desc>
-    <desc compset="_ELM%[^_]*CRDCTCBC"   >alm1.0 C only, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNRDCTCBC"  >alm1.0 C-N, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNPRDCTCBC" >alm1.0 C-N-P, nutirent competition via relative demand, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNECACTCBC" >alm1.0 C-N, nutirent competition via equilibrium chemistry approximation, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM%[^_]*CNPECACTCBC">alm1.0 C-N-P, nutirent competition via equilibrium chemistry approximation, ctc soil cascade with black carbon deposition:</desc>
-    <desc compset="_ELM50"               >clm5.0 physics:</desc>
-    <desc compset="_ELM50%[^_]*SP"       >clm5.0 Satellite phenology:</desc>
-    <desc compset="_ELM50%[^_]*BGC"      >clm5.0 bgc (cn and methane):</desc>
+    <desc compset="_ELM"               >elm physics:</desc>
+    <desc compset="_ELM%[^_]*SP"       >elm Satellite phenology:</desc>
```

Comment:
These are additive with each one that matches in order so don't put "elm" in any of the others after "elm physics:".  If you had ELM%SP, the resulting string would be "elm physics: elm Satellite phenology:" with what you have now.

Reply:
> These are additive with each one that matches in order so don't put "elm" in any of the others after "elm physics:". If you had ELM%SP, the resulting string would be "elm physics: elm Satellite phenology:" with what you have now.

I think it's ok to have this redundancy. the elm physics and elm satellite phenology are very distinct features.